Engine/TestPlanPauseEventArgs.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace OpenTap;
+
+/// <summary>
+/// Event args when a pause has been requested.
+/// </summary>
+public sealed class TestPlanPauseEventArgs
```

Comment:
I'm wondering if the test plan start args should be available in this as well, but I'm okay with not including it